src/modules/call/call.controller.ts

```diff
@@ -120,7 +120,7 @@ export class CallController {
   ): Promise<JoinCallResponseDto> {
     const { uuid, email } = user || {};
     const isUserAnonymous =
-      (!joinCallDto?.anonymousId || joinCallDto?.anonymous) ?? !user;
+      (!!joinCallDto?.anonymousId || joinCallDto?.anonymous) ?? !user;
```

Comment:
@apsantiso Check that this condition matches what we want. I changed it because I think it was wrong
`!joinCallDto?.anonymousId` -> `!!joinCallDto?.anonymousId`